.github/workflows/pre-commit.yml

```diff
@@ -30,3 +30,5 @@ jobs:
           fetch-depth: 2
       - name: Check license
         run: ./check_license.sh
+      - name: Build the site image
+        run: docker build -t yunikorn/yunikorn-website:2.0.0 . --build-arg NODE_VERSION=16.14
```

Comment:
We should use the same construct for building as we do in the push build. We read the node version from the `.nvmrc` file. See the push flow lines 34 and 36 on the how and what.